tests/structuredheadings/formatcombo.js

```diff
@@ -142,6 +144,77 @@
             "applied pre to h1"
         );
       });
+    },
+
+    "autonumbered heading is undoable": function () {
```

Comment:
I first made separate tests for undo, but then started just adding undo checks in where it made sense.  This is the separate tests case